CHANGELOG.md

```diff
@@ -30,6 +30,22 @@ Semantic versioning in our case means:
 - Fixes `WPS115` false-positive on `StrEnum`, `IntEnum`, `IntFlag` attributes, #3381
 - Fixes `WPS432`, now it ignores magic numbers in `Literal`, #3397
 
+
+## 1.1.1
```

Comment:
Let's add it to the latest WIP version: 1.2.0
I can't support multiple bugfix releases realistically.